keyboards/wheatfield/split75/keymaps/art/keymap.c

```diff
@@ -1,64 +1,48 @@
-#include <art.h>
-#include <sendstring_workman_zxcvm.h>
-
-bool is_win = false;
+#include "art.h"
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

Reply:
Just realised I forgot to respond to this conversation. This change is done